spellchecker/symspell.py

```diff
@@ -154,43 +154,41 @@ def get_canonical_term(key: str, former_canonical_term: str, canonical_term: str
 
         # create deletes
         edits = self.edits_prefix(key)
+
         for delete in edits:
             delete_hash = self.get_str_hash(delete)
             self._deletes[delete_hash].append(key)
         return True
 
     def load_dictionary(self, word_iterator: Iterable[tuple]):
         """
-        Load multiple dictionary entries from a file of word/frequency
-        count pairs. Merges with any dictionary data already loaded.
+        Load multiple dictionary entries from a file of word/frequency count pairs. Merges with any dictionary data
```

Comment:
I forgot to update the description to word, count and canonical word. Can you please adapt it?

Reply:
No problem